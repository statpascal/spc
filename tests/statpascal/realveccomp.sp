program realveccomp;

const
    n = 20;

var 
    a: vector of real;

begin
    a := intvec (1, n);

    writeln (a);
    writeln ('<  ', a < 10);
    writeln ('>  ', a > 10);
    writeln ('=  ', a = 10);
    writeln ('<> ', a <> 10);
    writeln ('<= ', a <= 10);
    writeln ('>= ', a >= 10)
end.
