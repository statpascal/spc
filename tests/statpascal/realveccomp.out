1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 
<  TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE 
>  FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE 
=  FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE 
<> TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE FALSE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE 
<= TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE 
>= FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE FALSE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE TRUE 
